docs/tutorials/basic/python.md

```diff
@@ -104,12 +104,18 @@ message Point {
 
 ## Generating client and server code
 
-Next you need to generate the gRPC client and server interfaces from your .proto service definition. You do this using the protocol buffer compiler `protoc` with a special gRPC Python plugin. Make sure you've installed protoc and followed the gRPC Python plugin [installation instructions](https://github.com/grpc/grpc/blob/{{ site.data.config.grpc_release_branch }}/INSTALL.md) first):
+Next you need to generate the gRPC client and server interfaces from your .proto service definition.
 
-With `protoc` and the gRPC Python plugin installed, use the following command to generate the Python code:
+First, install the gRPC Protocol Buffer plugin:
```

Comment:
"the gRPC Protocol Buffer plugin", as we use the term, isn't the same thing as `grpcio-tools`. I think this text should say "First, install the `grpcio-tools` package" (or something like that).